playground/mocks/data/idp/idx/authenticator-enroll-phone.json

```diff
@@ -40,15 +40,16 @@
                         "value": "sms"
                       },
                       {
-                        "label": "VOICE",
+                        "label": "Voice call",
```

Comment:
For the changes here, created an issue - https://oktainc.atlassian.net/browse/OKTA-304448